### DISCUSSION TITLE
升级最新版2.15.4日志记录看不了，新建的任务不能自动运行

### DISCUSSION THREAD

**Discussion by Jingle-Pan** [whyour/qinglong#1768] [Q&A]
![image](https://user-images.githubusercontent.com/75921710/210290346-9f8bf7b2-60fd-443b-a092-9de20cbb4f16.png)
如图，脚本运行了，在定时任务-操作-日志，看不到日志

**Comment by ireflux** [comment]
一样，我直接从 master 拉的最新的，本地启动，任务也不会自动运行（手动执行运行正常），不知道为啥，但日志可以看
